### PR TITLE
Add 2018 field and game data to sim

### PR DESCRIPTION
**Pull request by twinters007**
Fixes: #101
Fixes: #103

### CODE REVIEW

File: lib/pyfrc/mains/cli_sim.py [hunk 1]
```diff
@@ -14,11 +14,11 @@
 _field_root = abspath(join(dirname(__file__), '..', 'sim', 'field'))
 
 _field_defaults = {
```
Comment (virtuald) [inline lib/pyfrc/mains/cli_sim.py]: Don't delete the 2017 information, just add new information.

File: lib/pyfrc/sim/ui.py [hunk 2]
```diff
@@ -335,21 +335,35 @@ def _set_mode():
...
                                               lambda v: self.idle_add(self.on_auton_choices, v),
                                               lambda v: self.idle_add(self.on_auton_selection, v))
-            
+
             auton.pack(side=tk.TOP)
-        
+
+            gamedata = tk.LabelFrame(ctrl_frame, text='Game Data')
+
+            self.gamedatabox = Combobox(gamedata, state='readonly')
```
Comment (auscompgeek) [inline lib/pyfrc/sim/ui.py]: readonly?

Comment (auscompgeek) [reply lib/pyfrc/sim/ui.py]: Took a read of the tkinter.ttk docs. I think we want to future-proof this and let the user type arbitrary data (so keep the default state). Would rather not have users wait for us to update this next year if this is being used in future games.

Comment (twinters007) [reply lib/pyfrc/sim/ui.py]: Agreed

Comment (virtuald) [reply lib/pyfrc/sim/ui.py]: I think if you just get rid of the readonly thing now then we don't have to worry about it next year.

**Comment by twinters007** [comment]
I tested the hal data setting. It works. I might look at the code David mentioned in gitter and use the objects. I wanted to make the colors change when the different modes changed. For later though

**Comment by auscompgeek** [comment]
Wait. Typing in arbitrary game data doesn't actually work. Hm.

**Comment by twinters007** [comment]
Probably because it uses the on_selected method

**Comment by auscompgeek** [comment]
I think if you change the selection mode, it will work.

**Comment by virtuald** [comment]
Fixed the entry problem... Tk sucks. Once the tests pass I'll merge.

**Comment by virtuald** [comment]
Actually, tests aren't affected by this.

**Pull request merged**